### PR TITLE
Simplify client's logger configuration in tests.

### PR DESCRIPTION
**Pull request by ptabor**

### CODE REVIEW

File: client/v3/client_test.go [hunk 1]
```diff
@@ -30,7 +30,9 @@ import (
 )
 
 func NewClient(t *testing.T, cfg Config) (*Client, error) {
-	cfg.Logger = zaptest.NewLogger(t)
+	if cfg.Logger == nil {
+		cfg.Logger = zaptest.NewLogger(t)
```
Comment (lilic) [inline client/v3/client_test.go:34]: Why do we want a named client in the `tests/integration/testing.go` but not here?

**Review by lilic — APPROVED** [review]
One question, otherwise lgtm 
Thank you!

File: client/v3/client_test.go (hunk 1 above)
Comment (ptabor) [reply client/v3/client_test.go:34]: Fixed.

Comment (lilic) [reply client/v3/client_test.go]: Thanks!

**Pull request merged**